components/typography/styles.js

```diff
@@ -10,6 +10,15 @@ export const bodyStandardStyles = css`
 		bottom: 0.35rem;
 		top: 0.3rem;
 	}
+	:host([skeleton]) .d2l-body-standard.d2l-skeletize-paragraph-2 {
```

Comment:
Just spit-balling here.  If it's a component, it would be possible to have a little logic to go with some style + var.

``` css
:host {
    --d2l-skeletize-lines: 2;
    max-height: calc(1.4rem * var(--d2l-skeletize-lines));
}
```

``` javascript
updated(changedProperties) {
    changedProperties.forEach((_, propName) => {
        if (propName === 'lines') {
            let lines = parseInt(this.lines);
            if (isNaN(lines)) lines = ???;
            this.style.setProperty('--lines', `${--d2l-skeletize-lines}`);
        }
    });
}
```
While I like the dynamic/flexible nature of this, I'm not sure I like the overhead (which could be minuscule).

Reply:
The problem is the `1.4rem` varies based both on the typography style (standard, compact, small) but also changes at a responsive breakpoint. For example, "standard" is `1.4rem` normal and `1.2rem` on mobile, "compact` is always `1.2rem` and "small" is `1rem` normal and `0.9rem` on mobile.

If we were going to do a component, I think I'd just make it `<d2l-skeleton-paragraph type="compact" lines="9">` and then inside its rendering I'd loop X times outputting `<span class="d2l-body-compact d2l-skeletize"></span>`. Its height would then match up perfectly and there'd be no need for a `max-height`.

Reply:
The `max-height` is only needed because we're applying this background image to an existing element which may have a bunch of stuff in it and we want to restrict its height to line up exactly with the height of X lines of skeleton.